Tests/DatadogTests/Datadog/TracerTests.swift

```diff
@@ -583,7 +583,7 @@ class TracerTests: XCTestCase {
         )
         XCTAssertEqual(
             try spanMatcher.meta.custom(keyPath: "meta.person"),
-            #"{"age":30,"name":"Adam","nationality":"Polish"}"#
+            #"{"name":"Adam","age":30,"nationality":"Polish"}"#
```

Comment:
Is it required because of the change in `JSONEncoder`? If so - are we sure it's not flaky?

Reply:
Yeah this was added in #323 along with the key-sorting for `JSONEncoder`:
https://github.com/DataDog/dd-sdk-ios/pull/323/files#diff-25ffd9e630ae2d8dc218a66fa74cdb1ccf96171e92a96b13cc7270d7c9a7b77f

So after reverting the `JSONEncoder` setup, this test should fail consistently if we don't revert that line as well.